File: cap.10_linguagem_c++/codigos/exercicios/SOMA_IMPARES.cpp
#include <iostream>

using namespace std;

int main() {
    int num1, num2, soma, troca = 0;
    
    cout << "Digite dois numeros:" << endl;
    cin >> num1 >> num2;

    if (num1 > num2) {
        troca = num1;
        num1 = num2;
        num2 = troca;
    }

    for (int i = num1+1; i < num2; i++) {
        if (i % 2 != 0) {
            soma = soma + i;
        }
    }
    
    cout << "SOMA DOS IMPARES = " << soma;

    return 0;
}

File: cap.11_java/codigos/exercicios/src/SomaImpares.java
import java.util.Locale;
import java.util.Scanner;

public class SomaImpares {

    public static void main(String[] args) {
        
        Locale.setDefault(Locale.US);
        Scanner sc = new Scanner(System.in);

        int num1, num2, troca, soma = 0;

        System.out.println("Digite dois numeros:");
        num1 = sc.nextInt();
        num2 = sc.nextInt();

        if (num1 > num2) {
            troca = num1;
            num1 = num2;
            num2 = troca;
        }

        for (int i = num1+1; i < num2; i++) {
            if (i % 2 != 0) {
                soma = soma + i;
            }
        }

        System.out.println("SOMA DOS IMPARES = " + soma);

        sc.close();
    }

}
